module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -849,6 +873,25 @@ public function getHolding($bibId, array $patron = null, array $options = [])
                 }
                 $nextBatch[] = $nextItem;
             }
+
+            // If there are no item records at this location, we're going to create a fake one,
```

Comment:
Technically this is "if there are no item records on this holdings record".  Often one holdings record per location, but there is nothing mandating that.

Reply:
Good friendly amendment...I'll update the comment.